SpotifyAPI/Web/SpotifyWebAPI.cs

```diff
@@ -44,6 +44,25 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        #region Configuration
+        /// <summary>
+        /// Specifies after how many miliseconds should a failed request be retried.
+        /// </summary>
+        public int RetryAfter { get; set; } = 50;
+
+        /// <summary>
+        /// Should a failed request (Error 500, 502, or 503) be automatically retried or not.
```

Comment:
A reference to the `RetryErrorCodes`-property would be better than `500, 502 etc.`